version.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package yarpc
+
+// Version must be updated manually.
```

Comment:
mmm good point. // Version is the version. Wow details, very much useful.
stupid lint requirement. any good idea for the comment?

On Tue, Sep 13, 2016 at 5:56 PM, Abhinav Gupta <notifications@github.com>
wrote:

> In version.go
> <https://github.com/yarpc/yarpc-go/pull/331#discussion_r78671671>:
>
> > +// furnished to do so, subject to the following conditions:
> > +//
> > +// The above copyright notice and this permission notice shall be included in
> > +// all copies or substantial portions of the Software.
> > +//
> > +// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
> > +// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
> > +// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
> > +// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
> > +// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
> > +// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
> > +// THE SOFTWARE.
> > +
> > +package yarpc
> > +
> > +// Version must be updated manually.
>
> This will show up in the godoc. Probably not what you want for this
> message.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/yarpc/yarpc-go/pull/331/files/52a994ee5b452d104882b5fda02f58eb5a265e04#r78671671>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANMjDhUGYAUzGiyHUCJvbG1atHUrDTYks5qp0ZRgaJpZM4J8R0d>
> .
>